app/konnect/updates.md

```diff
@@ -11,6 +11,13 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## November 2022
 
+### 2022.11.21
+
+**Application Registration support in any runtime group**
+: {{site.konnect_short_name}} now officially supports the [app registration to services in both default and non-default runtime groups](/konnect/dev-portal/applications/enable-app-reg/#support-for-any-runtime-group). Portal developers will be able to register the application to consume services proxied through gateway services in both default and non default runtime groups.
+
+Currently, this feature is only available for {{site.base_gateway}} 3.0.0.0 or later.
```

Comment:
```suggestion
:  Currently, this feature is only available for {{site.base_gateway}} 3.0.0.0 or later.
```

Reply:
(sorry, accidentally edited your edit, but that's my suggestion above)

"Currently" implies that this is going to change. Is that the case?